modules/sharedaddy/sharing-sources.php

```diff
@@ -409,14 +409,18 @@ public function js_dialog( $name, $params = array() ) {
 		// Add JS after sharing-js has been enqueued.
 		wp_add_inline_script( 'sharing-js',
 			"var windowOpen;
-			jQuery( document.body ).on( 'click', 'a.share-$name', function() {
-				// If there's another sharing window open, close it.
-				if ( 'undefined' !== typeof windowOpen ) {
-					windowOpen.close();
+			document.body.addEventListener( 'click', function ( event ) {
+				if ( event.target &&
+						( event.target.matches && event.target.matches( 'a.share-$name' ) ) ||
+						( event.target.msMatchesSelector && event.target.msMatchesSelector( 'a.share-$name' ) ) ) {
```

Comment:
This doesn't seem right, the `event.target` guard is only applying to the first part even though it's used in the second.
```suggestion
				if ( event.target && (
					event.target.matches && event.target.matches( 'a.share-$name' ) ||
					event.target.msMatchesSelector && event.target.msMatchesSelector( 'a.share-$name' )
				) ) {
```

Reply:
Excellent catch on those parens, @anomiex, thank you! 👍